tests/integration/helm_opentelemetry_operator_enabled_test.go

```diff
@@ -0,0 +1,170 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	otoperatorappsv1 "github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
+)
+
+func Test_Helm_OpenTelemetry_Operator_Enabled(t *testing.T) {
+	const (
+		tickDuration = time.Second
+		waitDuration = time.Minute * 2
+	)
+
+	// It is required to add v1alpha1 OT Operator Scheme to K8s Scheme
+	// https://github.com/open-telemetry/opentelemetry-operator/issues/772
+	if err := otoperatorappsv1.AddToScheme(scheme.Scheme); err != nil {
+		assert.Fail(t, "failed to register scheme: %v", err)
```

Comment:
```suggestion
		require.Fail(t, "failed to register scheme: %v", err)
```

No point in continuing the test if that fails, right?